packages-web/calendar/tests/e2e/specs/renderDifferentViews.spec.ts

```diff
@@ -0,0 +1,26 @@
+import page from "../../../../../configs/e2e/src/pages/page";
+import Calendar from "../objects/calendar.widget";
+
+describe("Calendar", () => {
+    const calendar = new Calendar("calendar1");
+
+    it("renders month view", () => {
+        page.open();
+
+        calendar.element.waitForExist();
+        calendar.monthView.waitForExist();
+
+        expect(calendar.monthView.isExisting).toBeTruthy();
```

Comment:
This is already checked in the previous statement. Maybe you can do something similar with days displayed...

Reply:
i know, but this is the test